### PR TITLE
[#1981] Modify EditorHelper to be IE9 friendly

### PR DESCRIPTION
**Pull request by cadecairos**
https://webmademovies.lighthouseapp.com/projects/65733-popcorn-maker/tickets/1981

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @stanica received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/0a39189e7d25a60/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/0a39189e7d25a60/output.txt

Total script time: 0.92 mins

Server is running on http://pr837.butyr.org/templates/basic/
Run the Test Suite: http://pr837.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/a1699e14cfbfbf6/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/a1699e14cfbfbf6/output.txt

Total script time: 0.90 mins

Server is running on http://pr837.butyr.org/templates/basic/
Run the Test Suite: http://pr837.butyr.org/test/

File: templates/assets/editors/editorhelper.js [hunk 1]
```diff
@@ -91,6 +23,74 @@
     butter.listen( "trackeventupdated", _updateFunction );
   };
 
+  /**
+   * Member: draggable
+   *
+   * Makes a container draggable using jQueryUI
+   *
+   * @paran {TrackEvent} trackEvent: The trackEvent to update when position changes
```
Comment (mjschranz) [inline templates/assets/editors/editorhelper.js]: param please

File: templates/assets/editors/editorhelper.js [hunk 2]
```diff
@@ -91,6 +23,74 @@
...
+      }
+    });
+  };
+
+  /**
+   * Member: resizable
+   *
+   * Makes a container resizable using jQueryUI
+   *
+   * @paran {TrackEvent} trackEvent: The trackEvent to update when size changes
```
Comment (mjschranz) [inline templates/assets/editors/editorhelper.js]: param please

**Pull request closed without merging**